portfolio/src/main/webapp/script.js

```diff
@@ -90,3 +90,11 @@ async function getContent() {
   const quote = await response.text();
   document.getElementById('quote-container').innerText = quote;
 }
+function getJsonData() {
```

Comment:
if you are interested; this would be a great time to use the async syntax:

async function getJsonData() {
   const jsonPromise = await(response).json();
   const stringData = await jsonPromise;
   for (const comment of ...) { .. .}
}